src/include/daos_uns.h

```diff
@@ -137,6 +187,16 @@ duns_destroy_path(daos_handle_t poh, const char *path);
 int
 duns_parse_attr(char *str, daos_size_t len, struct duns_attr_t *attr);
 
+/**
+ * Free internal buffers allocated by the DUNS on the \a attr struct.
+ *
+ * \param[in]	attr	Attr pointer that was passed in to duns_resolve_path.
```

Comment:
I've got a PR that will perform doxygen checks on these files but we need to resolve existing warnings first.
```suggestion
 * \param[in]	attrp	Attr pointer that was passed in to duns_resolve_path.
```